### ISSUE TITLE
[A11y]: Write accessibility statement

### ISSUE THREAD

**Issue by Sidnioulz** [storybookjs/storybook#33453]
### Problem statement

We need an accessility statement for WCAG compliance and to help our users with disabilities set expectations.

### M1: Write page
**Owner: @Sidnioulz 
- [ ] Write scaffolding of page using https://www.w3.org/WAI/planning/statements/
- [ ] List known limitations

### M2: Deploy page on specific route
**Owner: @Sidnioulz 
- [ ] Ensure page is reachable via a dedicated URL like settings/what's new/etc

### M3: Link to page
**Owner: @Sidnioulz 
- [ ] List page in the menu
- [ ] List page in hidden link at start of website
- [ ] Link to page in the about page

**Comment by Sidnioulz** [comment]
Draft
```html
<h2>Accessibility Statement for <span class="basic-information website-name">Storybook</span></h2>
<p>
	This is an accessibility statement from <span class="basic-information organization-name">Storybook</span>.
</p>
<h3>Measures to support accessibility</h3>
<p>
	<span class="basic-information organization-name">Storybook</span>
	takes the following measures to ensure accessibility of
	<span class="basic-information website-name">Storybook</span>:
</p>
<ul class="organizational-effort accessibility-measures">
	<li>Include accessibility throughout our internal policies.</li>
	<li>Appoint an accessibility officer and/or ombudsperson.</li>
	<li>Employ formal accessibility quality assurance methods.</li>
	<li>Request review from external accessibility experts</li>
	<li>Use automated accessibility testing to monitor new issues</li>
</ul>
<h3>Conformance status</h3>
<p>
	The <a href="https://www.w3.org/WAI/standards-guidelines/wcag/">Web Content Accessibility Guidelines (WCAG)</a> defines requirements for designers and developers to improve accessibility for people with disabilities. It defines three levels of conformance: Level A, Level AA, and Level AAA.
	<span class="basic-information website-name">Storybook</span>
	is
	<span class="basic-information conformance-status" data-printfilter="lowercase">partially conformant</span>
	with
	<span class="basic-information conformance-standard"><span data-negate="">WCAG 2.2 level AA</span>.</span>
	<span>
	<span class="basic-information conformance-status">Partially conformant</span>
	means that
	<span class="basic-information conformance-meaning">some parts of the content do not fully conform to the accessibility standard</span>.
</span>
</p>
<h3>Feedback</h3>
<p>
	We welcome your feedback on the accessibility of
	<span class="basic-information website-name">Storybook</span>.
	Please let us know if you encounter accessibility barriers on
	<span class="basic-information website-name">Storybook</span>:
</p>
<ul class="basic-information feedback h-card">
	<li class="contact-other p-note">https://github.com/storybookjs/storybook/issues/new?template=bug_report.yml</li>
</ul>
<p>
	We try to respond to feedback within <span class="feedback responsetime">one to two weeks</span>.
</p>
<h3>Compatibility with browsers and assistive technology</h3>
<p>
	<span class="basic-information website-name">Storybook</span>
	is designed to be compatible with the following assistive technologies:
</p>
<ul class="technical-information compatible-environments">
	<li>Chrome with NVDA on Windows</li>
	<li>Firefox with NVDA on Windows</li>
	<li>Safari with VoiceOver on Mac</li>
</ul>
<p>
	<span class="basic-information website-name">Storybook</span>
	is not compatible with:
</p>
<ul class="technical-information incompatible-environments">
	<li>TODO: generate browserlist from the browserlist in package meta</li>
	<li>Narrator and JAWS screenreaders are not routinely tested against</li>
</ul>
<h3>Technical specifications</h3>
<p>
	Accessibility of
	<span class="basic-information website-name">Storybook</span>
	relies on the following technologies to work with the particular combination of web browser and any assistive technologies or plugins installed on your computer:
</p>
<ul class="technical-information technologies-used">
	<li>HTML</li>
	<li>WAI-ARIA</li>
	<li>CSS</li>
	<li>JavaScript</li>
</ul>
<p>These technologies are relied upon for conformance with the accessibility standards used.</p>
<h3>Limitations and alternatives</h3>
<p>
	Despite our best efforts to ensure accessibility of
	<span class="basic-information website-name">Storybook</span> , there may be some limitations. Below is a description of known limitations, and potential solutions. Please contact us if you observe an issue not listed below.
</p>
<p>
	Known limitations for
	<span class="basic-information website-name">Storybook</span>:
</p>
<ol class="technical-information accessibility-limitations">
	<li><strong>Sidebar navigation</strong>: Keyboard navigation in the sidebar is not accessible. because The Storybook sidebar is particularly complex and does not have an equivalent semantic HTML element that supports all of its features.. We are testing an alternative design of the sidebar that can work with all screenreaders. If you use search to find specific stories, press the <kbd aria-label="Arrow Up">↑</kbd> and <kbd aria-label="Arrow Down">↓</kbd> keys from the search input to navigate to search results.If you use the sidebar directly, press <kbd aria-label="Tab">⭾</kbd> to navigate between content sections, and press <kbd aria-label="Arrow Up">↑</kbd> and <kbd aria-label="Arrow Down">↓</kbd> to enter a content section when on the title of a section. Each section is immediately followed by a button to expand all items within it. We recommend expanding all items before entering the section, as expanding and collapsing is not currently possible with keyboard navigation..</li>
	<li><strong>Storybook Test widget does not announce when tests are done running</strong>: The test widget at the bottom of the sidebar allows you to run tests on all stories, but it does not report when tests are done running. because . We will implement live announcements in a future update to address this issue..tell it takes 10-30 seconds on small instances and up to a few mins on very large ones. Tell users to return to widget are results are not yet live aria.</li>
	<li><strong>Interactive stories autoplaying</strong>: Interactive stories on Storybook play automatically as you navigate to them. This is harmful to users with vestibular disorders. because Storybook Test relies on the stories playing, so we have not provided a setting to disable autoplay.. We are working on a solution to disable autoplay based on user motion preferences and to handle the side effects of that on Storybook Test. In the meantime, please ask your Storybook administrator to follow the [workaround](https://github.blog/engineering/user-experience/how-to-make-storybook-interactions-respect-user-motion-preferences/) kindly provided by the Design System team at GitHub..</li>
	<li><strong>You can find a more comprehensive and up-to-date list of known issues on [this page](https://github.com/orgs/storybookjs/projects/23/views/1). </strong>:because . . .</li>
</ol>
<h3>Assessment approach</h3>
<p>
	<span class="basic-information organization-name">Storybook</span>
	assessed the accessibility of
	<span class="basic-information website-name">Storybook</span>
	by the following approaches:
</p>
<ul class="technical-information assessment-approaches">
	<li>Self-evaluation</li>
</ul>
<h3>Evaluation report</h3>
<p>
	An evaluation report for
	<span class="basic-information website-name">Storybook</span>
	is available at:
	<a class="technical-information related-evidence" href="https://github.com/orgs/storybookjs/projects/23/views/1">https://github.com/orgs/storybookjs/projects/23/views/1</a>.
</p>
<hr noshade="noshade">
<h3>Date</h3>
<p>
	This statement was created on
	<span class="basic-information statement-created-date">2 January 2026</span>
	using the <a href="https://www.w3.org/WAI/planning/statements/">W3C Accessibility Statement Generator Tool</a>.
</p>
```
